src/js/components/App.tsx

```diff
@@ -33,7 +34,7 @@ const App = (props: AppSettings | any): JSX.Element => {
       //
       //Determine which resources we are reading from
       //Read our local resources.js file And any external library resources (which are prioritized)
-      dispatch(overwriteSettings({ ...resources, ...props }));
+      dispatch(overwriteSettings({ ...cameroon, ...props }));
```

Comment:
@vaidotasp @SwampGuzzler Just flagging I swapped the config file to test this PR